ofnetBgp.go

```diff
@@ -625,3 +592,13 @@ func (self *OfnetBgp) InspectProto() (interface{}, error) {
 
 	return OfnetBgpInspect, nil
 }
+
+func (self *OfnetBgp) SetRouterInfo(uplink *PortInfo) {
+	log.Infof("Received Set Router info for %v \n", uplink)
+	if uplink != nil && len(uplink.MbrLinks) != 1 {
+		log.Errorf("OfnetBgp currently supports only one uplink interface. Num uplinks active: %d", len(uplink.ActiveLinks))
+		return
```

Comment:
OK. This a new change I was not aware of. Thanks for the info.

But still, this seems to be triggered only during fwd-mode change. When the netplugin comes up for the first time, don't we still need to handle the scenario? Or, does netplugin start off with bridge mode and then switch to routing mode instantaneously (in which case we should be fine)?